<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaintainceDatetimesetPage</class>
 <widget class="QWidget" name="MaintainceDatetimesetPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>686</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color:black;
}

QLabel
{
	color:white;
	font:28px &quot;微软雅黑&quot;
}
QPushButton
{
	color:white;
	font:28px &quot;微软雅黑&quot;
}</string>
  </property>
  <widget class="QPushButton" name="returnBtn">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>621</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	font: 20px, &quot;微软雅黑&quot;;
    color: white;
	background-color: rgb(0, 128, 0);
    border-top: 2px solid white;
	border-left: 2px solid white;

}</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>611</y>
     <width>1018</width>
     <height>72</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(0, 0, 0);

	border:1px solid white;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="YearBtn">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>80</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;
color:rgb(30,200,200);
border:2px  solid white;
</string>
   </property>
   <property name="text">
    <string>123</string>
   </property>
  </widget>
  <widget class="QPushButton" name="MonthBtn">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>80</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;
color:rgb(30,200,200);
border:2px  solid white</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="DayBtn">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>80</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;
color:rgb(30,200,200);
border:2px  solid white</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="HourBtn">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>180</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;
color:rgb(30,200,200);
border:2px  solid white</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="SecondBtn">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>180</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;
color:rgb(30,200,200);
border:2px  solid white</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="MinuteBtn">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>180</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;
color:rgb(30,200,200);
border:2px  solid white</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>20</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>90</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>年</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>90</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>月</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>90</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>日</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>190</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>时</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>190</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>分</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>190</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>秒</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="ConfirmBtn">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>511</y>
     <width>270</width>
     <height>50</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px, &quot;微软雅黑&quot;;color: white;background-color: rgb(0, 128, 128);	border-radius:8px;border-top: 2px solid white;border-left: 2px solid white;</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>returnBtn</zorder>
  <zorder>YearBtn</zorder>
  <zorder>MonthBtn</zorder>
  <zorder>DayBtn</zorder>
  <zorder>HourBtn</zorder>
  <zorder>SecondBtn</zorder>
  <zorder>MinuteBtn</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>label_7</zorder>
  <zorder>label_8</zorder>
  <zorder>ConfirmBtn</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
